vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/GridTest.java

```diff
@@ -61,4 +61,36 @@ public void selectItem_lazyDataSet_selectionWorks() {
         Assert.assertTrue(grid.getSelectedItems().contains("foo"));
     }
 
+    @Test
+    public void make_all_rows_visible_by_setHeightByRows() {
```

Comment:
In Java you usually use camelCase for naming. The test naming convention is not super consistent in this repo, however at least in this class it seems to be `withPrecondition1_withPrecondition2_..._expectedResult`.

Reply:
done